themes/default/content/blog/deploy-aws-static-website-with-angular/index.md

```diff
@@ -0,0 +1,116 @@
+
+The great advantage of the AWS Static Website component is that you can have a complete, functioning site in a few minutes. The component encapsulates all the complex logic that an AWS static website requires. Without the component, you would need to spend hours or even days writing and debugging code to get the same result.
+
+If you’re interested in learning more about the component, read [Deploy Static Websites to AWS using 10 lines of YAML]({{< relref "/blog/deploy-aws-static-website-with-yaml" >}}).
+
+## The Angular Components
+
+All the Angular components you need to build the website are available in the GitHub [repository](https://github.com/sean1588/angular-static-site-pulumi). The repository contains:
+
+* A component for the [left navigation bar](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/shared/leftnav).
+* A component for the [top navigation bar](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/shared/topnav).
+* A component for the [Home](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/home) page.
+* A component for the [Menu](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/menu) page.
+
+We also use some components that come with the Angular framework.
+
+Each of the components has a TypeScript file that contains the programming logic and an HTML file that describes how the component will render.
+
+## Building the Pulumi Café Website
+
+Here are the instructions to build the website.
+
+1. Clone the GitHub repository located [here](https://github.com/sean1588/angular-static-site-pulumi).
+1. Run `npm install` to install the website’s node dependencies.
+1. Run `npm build` to generate the static website files and place them in a directory called **dist**. We will reference this directory when we configure Pulumi to deploy the site.
+
+## Deploy the Site with Pulumi
+
+You’re now ready to set up Pulumi to deploy the site to AWS.
+
+1. Create a directory named **pulumi** in the root of the project directory and then navigate to it with this command:
+
+    ```bash
+    mkdir pulumi && cd pulumi
+
+    ```
+
+1. You’re now in the **pulumi** directory. Generate a Pulumi project file and create a Pulumi.yaml file with this command:
+
+    ```bash
+    pulumi new aws-yaml
+    ```
+
+## Modify the .yaml File
+
+We need to add a few lines of code to the Pulumi.yaml file. In the `resources` section, the code references the AWS Static Website component and sets the **sitePath** property to the location of the **dist** build directory. We also want to know the URL of the site once it’s deployed. That’s what the `outputs` section  does. The **websiteURL** parameter is one of the outputs that the AWS Static Website component provides. Here is an example screenshot of the .yaml file.
```

Comment:
Little thing, but I'd probably instruct them to replace the contents of the file here:

```suggestion
We need to add a few lines of code to the Pulumi.yaml file. In the `resources` section, the code references the AWS Static Website component and sets the **sitePath** property to the location of the **dist** build directory. We also want to know the URL of the site once it’s deployed. That’s what the `outputs` section  does. The **websiteURL** parameter is one of the outputs that the AWS Static Website component provides. Replace the contents of `Pulumi.yaml` with the following code:
```